### DISCUSSION TITLE
double click to load sub in editor?

### DISCUSSION THREAD

**Discussion by Tsunder** [FakeFishGames/Barotrauma#9573] [Ideas and suggestions]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

wanted to try double clicking to load sub in the load sub menu instead of moving mouse all the way over and back (due to workshop sub warning which adds additional mouse movement)

the amount of mouse moving in the load sub is getting pretty big
would hugely want to stop game from warning me that every purple sub is a workshop sub.

### Reproduction steps

1. open edit
2. open load sub menu
3. double click a sub

### Bug prevalence

Happens every time I play

### Version

Other

### -

0.18.12

### Which operating system did you encounter this bug on?

Windows

**Comment by Ek-Videogames** [comment]
+1
I now make heavy use of the 'drag file into the window' feature exclusively to avoid having the dialog 20 times whenever I just need to load a humpback for mod & bug testing.

**Comment by MaaiG** [comment]
+1 Looks like we’re all thinking the same thing )). laziness and GUI stereotypes )))

> I now make heavy use of the 'drag file into the window'

Great! I didn’t know there was such a possibility!
